src/controllers/baseHandler.ts

```diff
@@ -65,9 +67,11 @@ export async function baseHandler(
       // data[1] is message obj
       const transformedArgument = {
         roleToBeTaggedObj: data[0],
-        displayMessageObj: data[1] ?? {},
+        displayMessageObj: data.find((item) => item.name === "message"),
+        channelId: message.channel_id,
+        dev: data.find((item) => item.name === "dev") as unknown as DevFlag,
```

Comment:
There was no other way I could manage this without actually overriding the type

Reply:
So to override the type we first need to override it and then specify the new type. Just like if a variable say `temp` 
 is first declared as of type string. Then I cannot just as  `temp as integer`. I would first need to tell the typescript to consider it as unknown and then only I can infer it as an integer.

Reply:
Created a separate issue to handle this
- https://github.com/Real-Dev-Squad/discord-slash-commands/issues/207